modules/gdscript/tests/scripts/completion/foo.notest.gd

```diff
@@ -0,0 +1,6 @@
+extends Node
+
+var test = 1
+
+func _ready():
+    $N￿
```

Comment:
This `0xFFFF` character might be annoying to add as there is no an easy way to insert it besides copying and pasting, and it's inconsistent how it's shown across editors. Though I can't think of a better alternative for this at the moment.

Reply:
We could choose some UTF8 emoji and replace it with 0xFFFF. The use of this emoji in the test would just be banned. Maybe ➡️ or 🖱️. But I'm not sure whether there could be problems because those emojis are longer then normal characters.